.github/semgrep/rule.yaml

```diff
@@ -0,0 +1,27 @@
+rules:
+  - id: jsx-not-internationalized
+    patterns:
+      - pattern: <$ELEMENT>$MESSAGE</$ELEMENT>
+      - metavariable-regex:
+          metavariable: $MESSAGE
+          regex: ([A-Za-z\n ]+[A-Za-z]+[A-Za-z\n ]+)
+      - pattern-not: <$ELEMENT>t('$KEY', ...)</$ELEMENT>
+    message: "JSX element not internationalized: '$MESSAGE'.  You should support
+      different languages in your website or app with internationalization.
+      Instead, use packages such as `i18next` in order to internationalize your
+      elements."
+    languages:
+      - typescript
+      - javascript
+    severity: WARNING
+    metadata:
+      category: portability
+      technology:
+        - react
+        - mui
+        - i18next
+      references:
+        - https://www.notion.so/hendyirawan/Internationalization-Localization-Policy-318c21674e5f44c48d6f136a6eb2e024
+        - https://mui.com/
+        - https://react.i18next.com/
+      license: Commons Clause License Condition v1.0[LGPL-2.1-only]
```

Comment:
Not sure if this metadata is necessary, if its referenced from blogpost or other place we can ignore this? specially around license as we are not 100% sure.

Reply:
I have removed the meta data side of things